src/components/MoneyRequestHeaderStatusBar.js

```diff
@@ -1,24 +1,35 @@
 import React from 'react';
 import {View} from 'react-native';
+import PropTypes from 'prop-types';
 import styles from '../styles/styles';
-import useLocalize from '../hooks/useLocalize';
 import Text from './Text';
 
-function MoneyRequestHeaderStatusBar() {
-    const {translate} = useLocalize();
+const propTypes = {
+    /** Title displayed in badge */
+    title: PropTypes.string.isRequired,
 
+    /** Banner Description */
+    description: PropTypes.string.isRequired,
+
+    /** Whether we show the border bottom */
+    shouldShowBorderBottom: PropTypes.bool.isRequired,
+};
+
+function MoneyRequestHeaderStatusBar({title, description, shouldShowBorderBottom}) {
```

Comment:
Heads up this broke the status bar in the split details page and I fixed it here https://github.com/Expensify/App/pull/29568/commits/fc1e3ff36f52a2cea32a66ab788b1d7a19a489f6